Snakefile

```diff
@@ -246,7 +247,6 @@ rule transphylo:
         startOff_p=config['transphylo']['startOff_p'],
         startPi=config['transphylo']['startPi'],
         optiStart=config['transphylo']['optiStart'],
-        dateT=config['transphylo']['dateT'],
```

Comment:
I think you do want to leave this in, and in the rule itself below (sorry if that was unclear), but just set a default of zero. In your R script, then if it's zero set it to max(dates) + a small amount as you originally did.

This is just because the default in the config is a bit too specific, but we still want to let people set it.